docs/testing.rst

```diff
@@ -50,3 +50,31 @@ Create your repository locally and upload that.
     $ scp -r * <fedora_username>:fedorapeople.org:/project/anaconda/repos/anaconda_fixup_repo
 
 After the above is done, everything should be fixed and Packit should work again.
+
+Fix ISO image refreshes for the WebUI tests
+-------------------------------------------
+
+In order to test the WebUI code we use the ISO images specified in `Cockpit's bots repository <https://github.com/cockpit-project/bots/tree/main/images>`_.
+
+These ISO files are direct downloads from the official distribution server and their
+`creation process is defined in bots repository <https://github.com/cockpit-project/bots/blob/main/images/scripts/fedora-rawhide-boot.bootstrap>`_.
+
+The purpose of using these ISOs for the tests instead of downloading from the official server is to gate the image updates.
+
+The default refresh period of the test images is one week, but they can also be `refreshed manually <https://github.com/cockpit-project/bots#refreshing-a-test-image>`_.
+
+Image refreshes happen with `Pull Requests which update the current image SHA <https://github.com/cockpit-project/bots/pull/2981>`_.
+The tests defined in `ui/webui/tests/` will run on the Pull request gating the `relevant to anaconda image refreshes <https://github.com/cockpit-project/bots/blob/main/lib/testmap.py>`_.
+
+In the case an updated dependency makes the WebUI tests on the image refresh fail, the on-duty team
+member is in charge of debugging the failure.
```

Comment:
Just to make sure, the on-duty team member does not have to manually confirm successful image updates, right ?

Reply:
Correct. That will be merged by the cockpit's on-duty person are part of the morning routine :) it's a manual process but they don't mind.

Reply:
Perfect, thanks for clarification. :)